src/pyspextool/io/check.py

```diff
@@ -102,16 +112,29 @@ def check_file(files:str):
         test = glob.glob(file)
         if not test:
 
-            message = 'File '+file+' not found.'
-            raise pySpextoolError(message)
+            if raise_error is True:
+
+                message = 'File '+file+' not found.'
+                raise pySpextoolError(message)
+
+            else:
+
+                return None
+
             
         else:
 
             if len(test) > 1:
 
-                message = 'More than one file matches '+file+'.'
-                raise pySpextoolError(message)
+                if raise_error is True:
+                
+                    message = 'More than one file matches '+file+'.'
+                    raise pySpextoolError(message)
+
+                else:
 
+                    return None
+                
```

Comment:
For the future, something to consider is adding a logging message when raise_error is False.  (I don't recall if we have the logger setup yet or not.)